docs/guides/tools-reference/file-transfer/comparing-data-transfer-utilities/index.md

```diff
@@ -0,0 +1,128 @@
+
+## What Is a Data Transfer Utility?
+
+A data transfer utility’s primary purpose is to facilitate the movement of data between different systems/devices, networks, and IT environments. At its core, a data transfer utility must transmit data from origin to destination efficiently and securely, with minimal disruption and errors. Depending on the use case, different data transfer tools may work better than others.
+
+Common data transfer scenarios range from simple file transfers and sharing to full-scale system backups, real-time data replication, and data migrations between different on-premise or cloud environments. Each use case often benefits from a different type of data transfer utility.
+
+## Transfer Considerations
+
+The following are some general transfer considerations to keep in mind when selecting a data transfer utility for a particular use case or scenario.
+
+### Egress and Ingress Cost
+
+Copying a single file from a desktop client to a cloud server calls for a different data transfer approach than migrating large volumes of data at scale between enterprise IT environments. When evaluating data transfer utilities, you should consider relative egress and ingress costs and the volume of data being transferred.
+
+### Data Storage Size and Type
+
+Along with how much data is being transferred, the kind of data and the destination are equally important (e.g., your data’s source and target destination, their respective file systems, and how much storage is required once the data is at rest).
+
+For example, most end users only deal with traditionally structured file systems when using their local environment, with data organized hierarchically into folders. In contrast, block storage breaks up files into separate blocks and stores them separately, allowing for significant performance benefits. Object storage saves files in a non-hierarchical, self-contained “flat” format. This allows for increased scalability and sizable reductions in cost and complexity.
+
+### File Type and Size
+
+Different file types and sizes also call for different data transfer utilities. For example, server log files grow in size quickly but can be easily compressed, whereas media files (e.g., MP4 video files, WAV sounds files) are usually quite large, aren’t easily compressed, and typically do not change once saved. Your choice of data transfer utility will depend on what kind of files are being transferred.
+
+Different file types also involve varying metadata considerations. Object storage is self-contained, so it includes both data and metadata for easy search and access and retrieval. Structured file system storage also includes extensive metadata; when right-clicking a file in your desktop operating system, you can view its create/modify time, file type, version number, and more. Alternatively, block storage is highly performant but provides no metadata capabilities.
```

Comment:
Similar to the last comment, I don't think block storage is an alternative to file systems--they go together. Will also take a swing at editing this in a bit